src/partition/repart.c

```diff
@@ -4077,6 +4077,12 @@ static int make_copy_files_denylist(Context *context, const Partition *p, Set **
                 }
         }
 
+        FOREACH_STRING(s, "proc", "sys", "dev", "run", "tmp", "var/tmp") {
+                r = exclude_children(s, &denylist);
```

Comment:
hmm, this looks wrong. In particular contents of /proc/ will change all the time, hence trying to generate a stable denylist from it is doomed to fail. This will break so hard in CI where processes might come and go in the background all the time.